db/seeds.rb

```diff
@@ -32,3 +32,64 @@
   status: 'active',
   working_pattern: 1
 )
+
+LocalDivisionalUnit.find_or_create_by!(
+    code: "WELDU",
+    name: "Welsh LDU",
+    email_address: "WalesNPS@example.com"
+)
+
+LocalDivisionalUnit.find_or_create_by!(
+    code: "ENLDU",
+    name: "English LDU",
+    email_address: "EnglishNPS@example.com"
+)
+
+LocalDivisionalUnit.find_or_create_by!(
+    code: "OTHERLDU",
+    name: "English LDU 2",
+    email_address: nil
+)
+
+# The offenders below are those with release dates a few years in the future and can therefore use the
+# responsibility override workflow
+
+CaseInformation.find_or_create_by!(
+    nomis_offender_id: 'G7658UL',
+    tier: 'A',
+    case_allocation: 'NPS',
+    welsh_offender: 'Yes',
+    manual_entry: true,
+    local_divisional_unit_id: 1,
```

Comment:
Rather than assume id of 1, might it be best to use the result of find_or_create_by! above to use that id?

Reply:
Implemented